duo_wordpress.php

```diff
@@ -156,6 +156,11 @@ function duo_auth_enabled(){
             return false; //allows the XML-RPC protocol for remote publishing
         }
 
+        if (defined('REST_REQUEST') && REST_REQUEST && duo_get_option('duo_rest', '') == 'off') {
```

Comment:
@matgargano Rest assured, I'm planning to update this PR myself. Just making comments for my own reference as I read through. :)